packages/integrations/components/googleTagManager/index.js

```diff
@@ -54,8 +41,7 @@ const GoogleTagManager = (props) => {
           {`
             window.dataLayer = window.dataLayer || [];
             if (${isNode()}) {
-              var data = ${JSON.stringify(dataLayer)};
-              data.event = 'irving.initialRender';
+              var data = { event: 'irving.initialRender' };
```

Comment:
Leaving this for now but I wasn't able to get this event to fire. Possibly only fires in production hosting context?